crates/ruff_linter/src/rules/perflint/snapshots/ruff_linter__rules__perflint__tests__PERF401_PERF401.py.snap

```diff
@@ -1,34 +1,121 @@
 ---
 source: crates/ruff_linter/src/rules/perflint/mod.rs
 ---
-PERF401.py:6:13: PERF401 Use a list comprehension to create a transformed list
+PERF401.py:6:13: PERF401 [*] Use a list comprehension to create a transformed list
   |
 4 |     for i in items:
 5 |         if i % 2:
 6 |             result.append(i)  # PERF401
   |             ^^^^^^^^^^^^^^^^ PERF401
   |
+  = help: Replace for loop with list comprehension
 
-PERF401.py:13:9: PERF401 Use a list comprehension to create a transformed list
+ℹ Unsafe fix
+1 1 | def f():
+2 2 |     items = [1, 2, 3, 4]
+3   |-    result = []
+4   |-    for i in items:
+5   |-        if i % 2:
+6   |-            result.append(i)  # PERF401
+  3 |+    result = [i for i in items if i % 2]
+  4 |+      # PERF401
```

Comment:
The comment handling is a bit weird. Is there anything we can do to improve it?

Reply:
I also just noticed that comments which are inside of the for loop get deleted, so I'll see if I can modify it so it keeps those and fixes the whitespace